haystack/file_converter/pdf.py

```diff
@@ -130,7 +130,7 @@ def _read_pdf(self, file_path: Path, layout: bool, encoding: str) -> List[str]:
             command = ["pdftotext", "-enc", encoding, "-layout", str(file_path), "-"]
         else:
             command = ["pdftotext", "-enc", encoding, str(file_path), "-"]
-        output = subprocess.run(command, stdout=subprocess.PIPE, shell=False)
+        output = subprocess.run(command, stdout=subprocess.PIPE, shell=False) # ignore
```

Comment:
Ignoring mypy check on line no #133 of pdf.py, not able to figure about the reason